src/NexusMods.DataModel/Synchronizer/SynchronizerService.cs

```diff
@@ -227,7 +226,7 @@ private Observable<LoadoutSynchronizerState> CreateStatusObservable(LoadoutId lo
                     if (busy)
                         return LoadoutSynchronizerState.Pending;
 
-                    if (last.Id != loadoutId)
+                    if (last.Id != loadoutId && last != default(LoadoutWithTxId))
```

Comment:
If the loadout is not the last one applied though, we do need to Apply. 

The problem is that we are in a state with no loadout applied.

This should currently happen while switching loadouts, and only very briefly on loadout creation. But it is somehow currently possible to get past loadout creation and still have no Last Applied loadout

Reply:
in that case it's handled further down b the "needs sync"